liota/core/package_manager.py

```diff
@@ -288,8 +288,23 @@ def _cmd_handler_list(self, parameter):
     #-----------------------------------------------------------------------
     # This method is used to handle statistical commands
 
-    def _cmd_handler_stat(self, parameter):
-        if parameter == "metrics" or parameter == "met":
+    def _cmd_handler_stat(self, parameters):
+        if parameters[0] == "package" or parameters[0] == "pkg":
+            if (len(parameters) < 2):
+                log.warning("package name is not specified")
+                return
+            if (len(parameters) > 2):
+                log.warning("Only 1 package name will be taken and processed")
+            query_pkg = parameters[1]
+            if query_pkg in self._packages_loaded.keys():
+                log.info("packages {0} is loaded".format(query_pkg))
+            else:
+                log.info("packages {0} is not loaded".format(query_pkg))
+            return
+        if len(parameters) != 1:
+            log.warning("Invalid format of stat command: %s" % parameters[0])
+            continue
```

Comment:
If it is an invalid format, why do we continue? Can't we just return?

Reply:
returned.